README.md

```diff
@@ -279,6 +279,41 @@ There are three different pipelines for demos use case: classification, instance
     make clean-results
     ```
 
+## pytorch object detection use case:
+
+Run the following commands by changing directory to $RETAIL_USE_CASE_ROOT/retail-use-cases/use-cases/demos
```

Comment:
```suggestion
Run the following commands by changing directory to ./retail-use-cases/use-cases/demos
```
when i am in the console, there is no env $RETAIL_USE_CASE_ROOT available, i think the best way is to set it to ./ right after the git clone

Reply:
this just refers to root directory for the project. I can change the name if its causing confusion.

Reply:
it does, if i just copied and pasted what documentation there, the RETAIL_USE_CASE_ROOT is not available in the console

Reply:
changed it 👍